CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### Added
+    - Issue 91 - Added Access-Control-Allow-Origin header to GET method
```

Comment:
Similar, we already logged the change for #91 so I think we can leave out a new changelog entry.

Reply:
Agree and removed!